platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingHashSessionIdManager.java

```diff
@@ -183,25 +188,21 @@ protected void provideNewSessionAttributes(
       String id,
       Map<String, Object> sessionAttributes) {
     // Make sure these attributes are different than the latest.
-    if (sessionAttributes != null && !sessionAttributes.equals(getLatestSessionAttributes(id))) {
+    if (sessionAttributes != null) {
       LOGGER.trace("Pushing new session attributes to all web contexts for session {}", id);
-      latestSessionAttributes.put(id, sessionAttributes);
       dataStores
           .stream()
           .filter(ds -> ds != callingDataStore)
```

Comment:
❓ Was the intent to exclude the calling data sotre by checking its identity and not equality?